configs/community.dbuild

```diff
@@ -1415,6 +1415,24 @@ build += {
     extra.test-tasks: "compile"
   }
 
+  ${vars.base} {
+    name: "scala-newtype"
+    uri:  ${vars.uris.scala-newtype-uri}
+    extra.exclude: ["newtypeJS", "catsTestsJS"]  // no Scala.js plz
+  }
+
+  ${vars.base} {
+    name: "fast-string-interpolator"
+    uri:  ${vars.uris.fast-string-interpolator-uri}
+    extra.sbt-version: ${vars.sbt-1-version}
+    extra.options: ["-Dbintray.user=dummy", "-Dbintray.pass=dummy"]
+    extra.commands: ${vars.default-commands} [
+      "set every bintrayReleaseOnPublish := false"
+    ]
+    // Missing dependency: com.dongxiguo#fastring
+    extra.exclude: ["benchmark"]
```

Comment:
I have isolated 3rd party interpolators in this module, to reproduce bug with macros compilation in `benchmark-core`: https://github.com/Sizmek/fast-string-interpolator/commit/346b61dbc3a00e623279f0faa21e307df73cc331

Reply:
thanks! I'll do at least one more run before merging, in order to test that change.